src/domain/ecoverse/ecoverse.resolver.queries.ts

```diff
@@ -118,10 +109,7 @@ export class EcoverseResolverQueries {
     });
   }
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.